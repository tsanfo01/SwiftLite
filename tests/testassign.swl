var x : String = "Not_Hello_World";
print(x);
x = "Hello_World";
print(x);