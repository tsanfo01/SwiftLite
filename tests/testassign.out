Not_Hello_World
Hello_World
